lib/cassanity/executors/cql_rb.rb

```diff
@@ -185,7 +184,7 @@ def call(args = {})
             end
 
             # Select the correct keyspace before executing the CQL query
-            if command != :keyspace_create && (keyspace_name = arguments[:keyspace_name])
+            if !command.to_s['keyspace'] && (keyspace_name = arguments[:keyspace_name])
```

Comment:
It looks like it would convert to string behind the scenes (http://ruby-doc.org/core-2.2.0/Symbol.html#method-i-match) and I can't seem to find anything different, can you confirm this?